src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -0,0 +1,186 @@
+    private readonly pythonVenvPathSetting: string = 'python.venvRelativePath';
+    private pythonAlias: string;
+    public getLaunchJson(): {} {
+        // currently the Python extension launches Windows with type: python and MacOS with type: pythonExperimental
+        const launchType: string = process.platform === Platform.Windows ? 'python' : 'pythonExperimental';
+        return {
+            version: '0.2.0',
+            configurations: [
+                {
+                    name: localize('azFunc.attachToJavaScriptFunc', 'Attach to Python Functions'),
+                    type: launchType,
+                    request: 'attach',
+                    port: 9091,
+                    host: 'localhost',
+                    preLaunchTask: funcHostTaskId
+                }
+            ]
+        };
+    }
+
+    public async addNonVSCodeFiles(): Promise<void> {
+        // need a custom message for warning as this is a dependency
+        if (await validateFuncCoreToolsInstalled(true)) {
+            await this.validatePythonVersion();
+            await this.ensureVirtualEnviornment();
+            await this.createPythonProject();
+        } else {
+            throw new UserCancelledError();
+        }
+    }
+
+    public async getTasksJson(): Promise<{}> {
+        return {
+            version: '2.0.0',
+            tasks: [
+                {
+                    label: localize('azFunc.runFuncHost', 'Run Functions Host'),
+                    identifier: funcHostTaskId,
+                    type: 'shell',
+                    dependsOn: 'build',
+                    osx: {
+                        command: `source ${await this.getVenvActivatePath(Platform.MacOS)} && func start host`
+                    },
+                    windows: {
+                        command: `${await this.getVenvActivatePath(Platform.Windows)} | func start host`
+                    },
+                    linux: {
+                        command: `source ${await this.getVenvActivatePath(Platform.MacOS)} && func start host`
+                    },
+                    isBackground: true,
+                    presentation: {
+                        reveal: 'always'
+                    },
+                    options: {
+                        env: {
+                            // tslint:disable-next-line:no-invalid-template-strings
+                            'languageWorkers:python:arguments': '-m ptvsd --server --port 9091 --file'
+                        }
+                    },
+                    problemMatcher: []
+                }
+            ]
+        };
+    }
+
+    private async validatePythonVersion(): Promise<void> {
+        const minReqVersion: string = '3.6.0';
+        let incompatiableVersion: boolean = false;
+        const pythonVersionIncompatiable: string = localize('pythonVersionIncompatiable', 'Current version of Python is incompatiable with Azure Functions Core Tools. Python 3.6 is required.');
```

Comment:
Reference `minReqVersion` in the string rather than hard-coding '3.6'. Otherwise we might forget to update the message even if we change `minReqVersion`